src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/CloudFoundryDeployerProperties.java

```diff
@@ -24,10 +24,13 @@
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
 /**
+ * Holds configuration properties for connecting to a Cloud Foundry runtime.
+ *
+ * @author Eric Bottard
  * @author Greg Turnquist
  */
 @ConfigurationProperties(prefix = "deployer.cf")
```

Comment:
Begging your pardon, but why not have "cloudfoundry" at the end, so any/all deployer configs would collect underneath "spring.cloud.deployer" in YAML?